src/NuGet.Clients/NuGet.PackageManagement.UI/Models/NuGetProjectUpgradeWindowModel.cs

```diff
@@ -64,134 +65,66 @@ private string ProjectName
 
         public IList<PackageDependencyInfo> PackageDependencyInfos { get; }
 
-        // Changing CollapseDependencies updates the list of included packages
-        public bool CollapseDependencies
-        {
-            get { return _collapseDependencies; }
-            set
-            {
-                _collapseDependencies = value;
-                OnPropertyChanged(nameof(DirectDependencies));
-                OnPropertyChanged(nameof(TransitiveDependencies));
-            }
-        }
-
-        public IEnumerable<string> Warnings
+        public IEnumerable<PackageIdentity> NotFoundPackages
```

Comment:
what is the purpose of this NotFoundPackages now?

Reply:
it skips uninstalling them.